Chatter/Controller/CreateAccountViewController.swift

```diff
@@ -95,7 +95,10 @@ class CreateAccountViewController: UIViewController {
     }
 
     @IBAction func chooseAvatarButtonPressed(_ sender: Any) {
-        performSegue(withIdentifier: Segues.toAvatarPicker.rawValue, sender: nil)
+        let viewModel = AvatarPickerViewModel()
+        let rootView = AvatarPickerRootView()
+        let viewController = AvatarPickerViewController(viewModel: viewModel, rootView: rootView)
+        present(viewController, animated: true, completion: nil)
```

Comment:
A suggestion for the future. You may want to use a flow controller to extract actual creating an MVVM for the avatar picker screen from this VC.